packages/mongodb-constants/src/stage-operators.ts

```diff
@@ -840,7 +840,7 @@ const STAGE_OPERATORS = [
   numCandidates: \${4:numCandidates},
   index: \${5:string},
   limit: \${6:limit},
-  filter: {\${}}
+  filter: {\${7:expression}}
```

Comment:
FWIW doesn't look like our tests are correct, this was a valid snippet (both for ace and codemirror)